RadixWallet/Clients/SubmitTransactionClient/SubmitTransactionClient+Interface.swift

```diff
@@ -50,17 +27,15 @@ public struct SubmitTXRequest: Sendable, Hashable {
 // MARK: - TransactionStatusUpdate
 public struct TransactionStatusUpdate: Sendable, Hashable {
 	public let txID: TXID
-	public let result: Result<GatewayAPI.TransactionStatus, TransactionPollingFailure>
+	public let result: Loadable<EqVoid>
 }
 
 // MARK: - PollStrategy
 public struct PollStrategy: Sendable, Hashable {
-	public let maxPollTries: Int
 	public let sleepDuration: TimeInterval
-	public init(maxPollTries: Int, sleepDuration: TimeInterval) {
-		self.maxPollTries = maxPollTries
+	public init(sleepDuration: TimeInterval) {
 		self.sleepDuration = sleepDuration
 	}
 
-	public static let `default` = Self(maxPollTries: 20, sleepDuration: 2)
+	public static let `default` = Self(sleepDuration: 2)
```

Comment:
have we checked with GW about this change? We should really impl incremental backing off strategy, perhaps as easy as adding 0.5 or 1 seconds to sleep duration after each poll attempt.

Reply:
yes, we do perform infinite polling, and stop only when we have certain transaction status returned.